lib/client-assets.php

```diff
@@ -440,6 +440,19 @@ function gutenberg_register_packages_styles( $styles ) {
 }
 add_action( 'wp_default_styles', 'gutenberg_register_packages_styles' );
 
+/**
+ * Gets the stylesheet handle for the current theme.
+ *
+ * @since 6.3
+ *
+ * retrn string
+ */
+function gutenberg_get_stylesheet_handle() {
```

Comment:
There may be a way to do this in WordPress already but I couldn't find it.

Reply:
I've been looking around as well, because it does seem like there's a better way. The only improvement I can find after searching for 10 minutes is to use `[get_stylesheet()](https://developer.wordpress.org/reference/functions/get_stylesheet/)`  instead of querying it directly from `get_option( 'stylesheet' )` just in case it has been filtered.

I'm trying to find where the stylesheet gets registered to see how the slug is built so we can replicate or use that.